html/janus.js

```diff
@@ -1772,23 +1772,36 @@ function Janus(gatewayCallbacks) {
 				pluginHandle.onremotestream(config.remoteStream);
 				if(event.track.onended)
 					return;
+				var trackMutedTimeoutId = null;
 				Janus.log("Adding onended callback to track:", event.track);
 				event.track.onended = function(ev) {
 					Janus.log("Remote track muted/removed:", ev);
 					if(config.remoteStream) {
-						config.remoteStream.removeTrack(ev.target);
-						pluginHandle.onremotestream(config.remoteStream);
+						clearTimeout(trackMutedTimeoutId);
```

Comment:
Shouldn't there be a `if(trackMutedTimeoutId != null) {` here as well?

Reply:
calling `clearTimeout(null)` just does nothing:
> Passing an invalid ID to clearTimeout() silently does nothing; no exception is thrown.

so I don't think is actually needed.